pkg/tuned/dialog.jsx

```diff
@@ -177,7 +177,7 @@ const TunedDialog = ({
                     /* Yup this is how tuned returns failures */
                         if (!results[0]) {
                             console.warn("Failed to disable tuned profile:", results);
-                            return Promise.reject(_("Failed to disabled tuned profile"));
+                            return Promise.reject(_("Failed to disable tuned profile"));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18021-20221212-113452-c079798e-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)